Source/GUI/dvrescue/dvrescue/CaptureView.qml

```diff
@@ -27,17 +27,24 @@ Column {
     property var csvParserUI: csvParserUI
 
     property int frameSpeed: 0
+
+    readonly property int thresholdBetweenFastPlayAndPlay: 35
+    readonly property int thresholdBetweenPlayAndSlowPlay: 30
+
     onFrameSpeedChanged: {
-        if(frameSpeed <= -50)
-            capturingModeInt = rewing
-        if(frameSpeed < 0)
-            capturingModeInt = srewing
-        if(frameSpeed === 0)
+        if(frameSpeed < 0) {
+            if(frameSpeed <= -thresholdBetweenFastPlayAndPlay)
+                capturingModeInt = rewing
+            else
+                capturingModeInt = srewing
+        } else if(frameSpeed > 0) {
+            if(frameSpeed >= thresholdBetweenFastAndPlay)

```

Comment:
typo? The variable `thresholdBetweenFastAndPlay` is only used once in this part but not defined.

Reply:
should be `thresholdBetweenFastPlayAndPlay`